content/applications/inventory_and_mrp/inventory/shipping/setup/bpost.rst

```diff
@@ -0,0 +1,105 @@
+=================
+bpost integration
+=================
+
+Set up the *bpost* shipping connector in Odoo to manage bpost shipments to clients directly within
+Odoo. To configure it, complete these steps:
+
+#. Create a bpost account.
+#. Get the :ref:`Account ID and passphrase <inventory/shipping/bpost-account>`.
+#. Set up the shipping method in Odoo.
+
+Upon completion, it is possible to calculate the cost of shipping based on package size and weight,
+have the charges applied directly to a bpost business account, and automatically print bpost
+tracking labels through Odoo.
+
+.. seealso::
+   - :doc:`third_party_shipper`
+   - :doc:`delivery_method`
+   - :doc:`dhl_credentials`
+   - :doc:`ups_credentials`
+
+bpost account setup
+===================
+
+To begin, go to the `bpost website <https://parcel.bpost.be/en/home/business>`_ to create or log
+into the company's bpost business account. When creating the bpost account, have the company's VAT
+number and mobile phone number ready. Follow the website's steps to complete registration and sign
+up for shipping services.
```

Comment:
```suggestion
To begin, go to the `bpost website <https://parcel.bpost.be/en/home/business>`_ to create, or log
into, the company's bpost business account. When creating the bpost account, have the company's VAT
number and mobile phone number ready. 

Follow the website's steps to complete registration, and sign up for shipping services.
```

SUGGESTION: added commas for enhanced sentence flow/clarity, broke up paragraph to appear more digestible to the reader